src/components/Footer/footer.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import style from "./footer.module.css";
+import image from "./sakura_footer.svg";
+
+const Footer = () => {
+  return (
+    <React.Fragment>
+      <footer className={style["footer"]}>
+        <div className={style["info-span"]}>
+        <img src={image} alt="img" height="60px" width="260px" className={style["footer-img"]}/>
+        <p>Project Sakura is a fully free and Opensource Custom ROM built around LineageOS.</p>
+        <p><div style={{color:"#347ab7"}}><b>EMAIL ADDRESS </b></div>
+            hello@reallygreatsite.com</p>
```

Comment:
use @LordShenron email here shen.priyanshu@gmail.com or I am not sure If we want to give email, Lets just remove the email line